### PR TITLE
[SDK] Added daily stats to ts sdk

### PR DESCRIPTION
**Pull request by eugenvoronov**
## Description
Created an subgraph entity dailyStats with the properties specified in the implementation details.

## Summary of changes
- [x] Implemented new properties for Typescript SDK
- [x] Updated unit tests for Typescript SDK
- [x] Implemented new properties for Python SDK
- [x] Updated unit tests for Python SDK
- [x] Updated subgraph schema

## How test the changes
**Typescript**
```typescript
import { StatisticsClient, ChainId, NETWORKS } from '@human-protocol/sdk';

const statisticsClient = new StatisticsClient(NETWORKS[ChainId.POLYGON_AMOY]);

// Fetch daily statistics data without specifying date range
const statsWithoutParams = await statisticsClient.getHMTStatistics();
console.log('HMT Statistics without Params:', statsWithoutParams);

// Fetch daily statistics data with specified date range
const from = new Date(2023, 4, 8);
const to = new Date(2023, 5, 8);
const statsWithParams = await statisticsClient.getHMTStatistics({ from, to });
console.log('HMT Statistics from 5/8 - 6/8:', statsWithParams);
```

`yarn test`

**Python**
```python
from human_protocol_sdk.contants import ChainId
from human_protocol_sdk.statistics import StatisticsClient, DailyStatsParam

statistics_client = StatisticsClient(ChainId.POLYGON_AMOY)

stats_without_params = await statistics_client.get_hmt_statistics()
print("HMT Statistics without Params:")
print(stats_without_params)

from = datetime.datetime(2024, 5, 8)
to = datetime.datetime(2023, 6, 8)
stats_with_params = await statistics_client.get_hmt_statistics(
    DailyStatsParam(from=from, to=to)
)
print("HMT Statistics with Params:")
print(stats_with_params)
```

`make unit-test`

## Related issues
[SDK] Daily statistics
[#2084](https://github.com/humanprotocol/human-protocol/issues/2084)

### CODE REVIEW

File: packages/sdk/typescript/subgraph/src/mapping/Escrow.ts [hunk 1]
```diff
@@ -234,6 +235,11 @@ export function handleBulkTransfer(event: BulkTransfer): void {
   eventDayData.dailyTotalEventCount =
     eventDayData.dailyTotalEventCount.plus(ONE_BI);
 
+  // Update daily stats
+  const dailyStats = getDailyStatsData(event);
+  dailyStats.escrowPayouts = dailyStats.escrowPayouts.plus(ONE_BI);
```
Comment (eugenvoronov) [inline packages/sdk/typescript/subgraph/src/mapping/Escrow.ts:240]: Works as a payout counter.

File: packages/sdk/python/human-protocol-sdk/human_protocol_sdk/statistics/statistics_client.py [hunk 2]
```diff
@@ -229,18 +229,25 @@ def __init__(
...
         :param total_transaction_amount: Total transaction amount
         :param total_transaction_count: Total transaction count
+        :param unique_senders: Total unique senders
+        :param unique_receivers: Total unique receivers
         """
 
         self.timestamp = timestamp
         self.total_transaction_amount = total_transaction_amount
         self.total_transaction_count = total_transaction_count
+        self.total_transaction_amount = total_transaction_amount
```
Comment (portuu3) [inline packages/sdk/python/human-protocol-sdk/human_protocol_sdk/statistics/statistics_client.py:248]: it is duplicated

File: packages/sdk/typescript/subgraph/src/mapping/utils/dayUpdates.ts [hunk 3]
```diff
@@ -27,6 +27,8 @@ export function getEventDayData(event: ethereum.Event): EventDayData {
     eventDayData.dailyPayoutAmount = ZERO_BI;
     eventDayData.dailyHMTTransferCount = ZERO_BI;
     eventDayData.dailyHMTTransferAmount = ZERO_BI;
+    eventDayData.uniqueSenders = ZERO_BI;
```
Comment (portuu3) [inline packages/sdk/typescript/subgraph/src/mapping/utils/dayUpdates.ts:30]: can you please rename to follow the same structure dailyUniqueSenders and dailyUniqueReceivers?

File: packages/sdk/typescript/subgraph/schema.graphql [hunk 4]
```diff
@@ -15,6 +15,18 @@ type Worker @entity {
   payoutCount: BigInt!
 }
 
+type UniqueSender @entity {
```
Comment (portuu3) [inline packages/sdk/typescript/subgraph/schema.graphql:18]: You don't need these new entities. You can check hmt transfer events of the day to know if the this wallet was already a sender or receiver.

**Review by portuu3 — CHANGES_REQUESTED** [review]

**Review by portuu3 — APPROVED** [review]

**Pull request merged**